applications/CoSimulationApplication/python_scripts/coupling_operations/distribute_point_values.py

```diff
@@ -1,38 +1,65 @@
 from __future__ import print_function, absolute_import, division  # makes these scripts backward compatible with python 2.6 and 2.7
 
+# Importing the Kratos Library
+import KratosMultiphysics as KM
+
 # Importing the base class
 from KratosMultiphysics.CoSimulationApplication.base_classes.co_simulation_coupling_operation import CoSimulationCouplingOperation
 
 class DistributePointValuesOperation(CoSimulationCouplingOperation):
-    # this is a dummy implementation, this should use the "VariableRedistributionUtility" from the core
-    def Initialize(self):
-        pass
-
-    def Finalize(self):
-        pass
-
-
-    def InitializeSolutionStep(self):
-        pass
 
-    def FinalizeSolutionStep(self):
-        pass
+    def __init__(self, settings, solver_wrappers):
+        super(ComputeNormalsOperation, self).__init__(settings)
+        solver_name = self.settings["solver"].GetString()
+        data_name = self.settings["data_name"].GetString()
+        self.interface_data = solver_wrappers[solver_name].GetInterfaceData(data_name)
 
+        self.redistribution_iterations = self.settings["redistribution_tolerance"].GetDouble()
+        self.redistribution_tolerance  = self.settings["redistribution_iterations"].GetInt()
```

Comment:
```suggestion
        self.redistribution_tolerance  = self.settings["redistribution_tolerance"].GetDouble()
```

Reply:
for testing the implementation, I added the required variables for the fluid model part (MAPPER_VECTOR_PROJECTION_RHS, NODAL_VAUX) inside the fluid solver, which is obviously not a good idea. So maybe one should think of a good place to add the extra variables to the fluid model part.